### ISSUE TITLE
Error building pest_consume 1.0.5

### ISSUE THREAD

**Issue by rachitnigam** [Nadrieril/pest_consume#7]
Steps to reproduce:
1. Clone the repository.
2. `cargo update`
3. `cargo build`

Error message:
```
    |
181 | pub use match_nodes::proc_macro_hack_match_nodes;
    |         ^^^^^^^^^^^^^---------------------------
    |         |            |
    |         |            help: a similar name exists in the module: `_proc_macro_hack_match_nodes`
    |         no `proc_macro_hack_match_nodes` in `match_nodes`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: could not compile `pest_consume`.
```

Version information:
```
⇒  rustc -V
rustc 1.47.0 (18bf6b4f0 2020-10-07)
⇒  cargo -V
cargo 1.47.0 (f3c7e066a 2020-08-28)
```

**Comment by Nadrieril** [comment]
What about not doing `cargo update`?

**Comment by Nadrieril** [comment]
That's the point of `Cargo.lock`: it fixes the versions of everything. If you update it then indeed it might break things

**Comment by rachitnigam** [comment]
The Github action for building our crate broke. As far as I can tell, the action doesn't seem to run cargo update? https://github.com/cucapra/futil/runs/1318613171?check_suite_focus=true

**Comment by Nadrieril** [comment]
Ah ok, it's a different issue then. I think it's because some other part of your project uses a more recent version of `proc_macro_hack`, and `proc_macro_hack` must have made a breaking change to whatever it is I'm using there

**Comment by Nadrieril** [comment]
Fixed in https://github.com/Nadrieril/pest_consume/commit/e3a020940c355a430df1631e9e54e11540f9b30f, released in 1.0.6

**Comment by Miezhiko** [comment]
Yes, issue has gone, thank you
